code/modules/surgery/organs/organ_external.dm

```diff
@@ -671,9 +671,9 @@ Note that amputating the affected organ does in fact remove the infection from t
 	if(!make_tough)
 		brute_mod = 0.66
 		burn_mod = 0.66		
-		if(owner && (ROBOTIC in owner.dna.species.species_traits))
-			return
 		dismember_at_max_damage = TRUE
+		if(owner && (ROBOTIC in owner.dna.species.species_traits))
+			dismember_at_max_damage = FALSE
```

Comment:
this will mean that it will only make limbs fall off at max damage if the robotic limb belongs to an IPC.

And instead of adding a new trait, why not make use of the `inherent_biotypes` flag?

Reply:
i plan on using the trait in future prs to modularize ipcs and their snowflakey behavior, i could remove it if needed

Reply:
also shouldn’t it make robotic limbs fall off at max damage unless it belongs to an ipc

Reply:
ah right I miss read. why do you set the variable then immediatelly override it. instead why don't you check `if(owner && !(ROBOTIC in owner.dna.species.species_traits)) -> dismember_at_max_damage = TRUE`